src/main/drivers/transponder_ir_erlt.c

```diff
@@ -34,7 +34,7 @@ void transponderIrInitERLT(transponder_t *transponder){
     transponder->vTable             = &erltTansponderVTable;
     transponder->timer_hz           = TRANSPONDER_TIMER_MHZ_ERLT;
     transponder->timer_carrier_hz   = TRANSPONDER_CARRIER_HZ_ERLT;
-    memset(&(transponder->transponderIrDMABuffer.erlt), 0, TRANSPONDER_DMA_BUFFER_SIZE_ERLT);
+    memset(&(transponder->transponderIrDMABuffer.erlt), 0, sizeof(transponder->transponderIrDMABuffer.arcitimer));
```

Comment:
I think this is unrelated to gcc. Should this go into a separate pull request? Also, is this define now unused and should it be removed?

Reply:
why this change is needed are descriped in the pull request message.
also of coure the marko is still in use. transponderIrDMABuffer uses this to determine the array size

Reply:
Still, it's a change that is unrelated to changing the gcc version, so it should be a separate pull request.